index.html

```diff
@@ -260,11 +260,12 @@ <h1>Contact me</h1>
                 I'd love to help with it </p>
             </div>
             <div class="contact-form">
-            <form action="https://formspree.io/html/" method="post">
+            <form action="https://formspree.io/f/xnqwordb" method="post">
                 <input name="first_name" id="text" type="text" maxlength="30" placeholder="Afolabi" required>
                 <input name="Email" id="email" type="email" placeholder="example@gmail.com" required>
-                <input type="text" id="message" maxlength="500" placeholder="write your message here" required>
-                <button type="submit" class="btn">Get It Touch</button>
+                <textarea id="message" name="message" maxlength="500" required>write your message here
+                </textarea>
+                <button type="submit" class="btn">Get in Touch</button>
```

Comment:
- [ ] Please increase the padding for button to increase the button size and make sure it matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166665066-8f7d3e67-5794-4d94-8ea0-829790a9ddb5.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166665101-d48abc2c-6996-4d62-9489-6966908e4e12.png)